bleak/__init__.py

```diff
@@ -8,13 +8,39 @@
 import os
 import sys
 import logging
-import platform
 import asyncio
 
 from bleak.__version__ import __version__  # noqa: F401
 from bleak.exc import BleakError
-
-_on_rtd = os.environ.get("READTHEDOCS") == "True"
+from bleak._api import (
+    BleakScanner,
+    _BleakScannerImplementation,
```

Comment:
To be consistent with the existing naming conventions, `_BleakXyzBackend` instead of `_BleakXyzImplementation` seems like the more obvious name.

Reply:
Agreed. Will do.